v20.1/admin-ui-cluster-overview-page.md

```diff
@@ -3,88 +3,131 @@ title: Cluster Overview Page
 
-When you decommission a node, CockroachDB lets the node finish in-flight requests, rejects any new requests, and transfers all range replicas and range leases off the node so that it can be safely shut down. See [Remove Nodes](remove-nodes.html) for more information.
+<img src="{{ 'images/v20.1/admin-ui-decommissioned-nodes.png' | relative_url }}" alt="CockroachDB Admin UI node list" style="border:1px solid #eee;max-width:100%" />
+
+{{site.data.alerts.callout_info}}
+When you [decommission a node](remove-nodes.html), CockroachDB lets the node finish in-flight requests, rejects any new requests, and transfers all range replicas and range leases off the node so that it can be safely shut down.
+{{site.data.alerts.end}}
 
 ## Node Map (Enterprise)
 
-The **Node Map** is an [enterprise-only](enterprise-licensing.html) feature that gives you a visual representation of the geographical configuration of your cluster.
+The **Node Map** is an [enterprise](enterprise-licensing.html) feature that visualizes the geographical configuration of your cluster. It requires that [`--locality` flags have been defined](cockroach-start.html#locality) for your nodes.
+
+For guidance on enabling and configuring the node map, see [Enable the Node Map](enable-node-map.html).
 
 <img src="{{ 'images/v20.1/admin-ui-node-map.png' | relative_url }}" alt="CockroachDB Admin UI Summary Panel" style="border:1px solid #eee;max-width:90%" />
 
-The Node Map consists of the following components:
+The Node Map is populated with **locality components** and **node components**, using the latitude and longitude of each locality to position the components on the map.
+
+### Locality component
+
+The map shows the components for the highest-level locality tier (e.g., region). You can click on the **Node Count** of a locality component to view any lower-level localities (e.g., availability zone). 
 
-### Region component
+For details on **Capacity Usage**, see [Capacity metrics](#capacity-metrics).
 
 <img src="{{ 'images/v20.1/admin-ui-region-component.png' | relative_url }}" alt="CockroachDB Admin UI Summary Panel" style="border:1px solid #eee;max-width:90%" />
 
 {{site.data.alerts.callout_info}}
-For multi-core systems, the user CPU percent can be greater than 100%. Full utilization of one core is considered as 100% CPU usage. If you have n cores, then the user CPU percent can range from 0% (indicating an idle system) to (n*100)% (indicating full utilization).
+On multi-core systems, the displayed CPU usage can be greater than 100%. Full utilization of 1 core is considered as 100% CPU usage. If you have _n_ cores, then CPU usage can range from 0% (indicating an idle system) to (_n_ * 100)% (indicating full utilization).
 {{site.data.alerts.end}}
 
 ### Node component
 
+Node components are accessed by clicking on the **Node Count** of the lowest-level [locality component](#locality-component). 
+
+For details on **Capacity Usage**, see [Capacity metrics](#capacity-metrics).
```

Comment:
Might be nice to give slightly more info e.g. "For details on how the capacity usage is calculated, see 'capacity metrics'"